docs/app/pages/charts/sections/bar-charts/horizontal-bar-chart/horizontal-bar-chart.component.html

```diff
@@ -0,0 +1,129 @@
+<div class="demo-chart-container">
+
+    <canvas baseChart 
+            [datasets]="barChartData"
+            [labels]="barChartLabels"
+            [options]="barChartOptions"
+            [legend]="barChartLegend"
+            [colors]="barChartColors"
+            chartType="horizontalBar">
+    </canvas>
+
```

Comment:
Looks a lot better than the old one :) Is there any way to have the x-axis line along the bottom instead of the top, with the labels?